src/pybind/mgr/influx/module.py

```diff
@@ -196,6 +204,19 @@ def send_to_influx(self):
         try:
             client.write_points(self.get_df_stats(), 'ms')
             client.write_points(self.get_daemon_stats(), 'ms')
+            self.set_health_checks(dict())
+        except ConnectionError as e:
+            self.log.exception("Failed to connect to Influx host %s:%d",
+                               self.config['hostname'], self.config['port'])
+            self.set_health_checks({
+                'MGR_INFLUX_SEND_FAILED': {
+                    'severity': 'warning',
+                    'summary': 'InfluxDB module could not contact server at '
+                               '%s:%d due to a ConnectionError' %
```

Comment:
let's say "connection error" instead of ConnectionError, as this string will go in front of non-programmer eyeballs